c7n/resources/rds.py

```diff
@@ -46,6 +46,7 @@
 import logging
 import re
 
+from pkg_resources import parse_version
```

Comment:
was there a reason for changing this import from distutils to pkg_resources? i'd prefer distutils which is a builtin, verse pkg_resources which is an add on (from setuptools).